app/src/main/java/se/inera/intyg/minaintyg/certificate/dto/CertificateResponseDTO.java

```diff
@@ -0,0 +1,13 @@
+package se.inera.intyg.minaintyg.certificate.dto;
+
+import lombok.Builder;
+import lombok.Value;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.CertificateMetadata;
+
+@Value
+@Builder
+public class CertificateResponseDTO {
```

Comment:
Could we return a formattedCertificate instead? The field can just be called certificate. 

It won't break compatibility if we get the need in the future to return some additional data that is not directly part of the formatted certificate...